public/assets/js/main.js

```diff
@@ -8,11 +8,9 @@
         var el = document.querySelectorAll(selector);
 
         function toggleCode(index) {
-            var name = el[index].getAttribute('data-el');
             var btnIcon = el[index].querySelector('span');
 
             el[index].addEventListener('click', function() {
-                document.getElementById(name).classList.toggle('hidden');
                 btnIcon.classList.toggle('caret-open');
```

Comment:
Maybe you can do : 

```javascript
setTimeout(function () {
  btnIcon.classList.toggle('caret-open');
}, 350);
```

`350` is the transition duration of the Collapse plugin (https://github.com/twbs/bootstrap/blob/v3-dev/js/collapse.js#L35)